src/isitcg.web/ingredientrules.json

```diff
@@ -23,7 +23,19 @@
                 "Ethyl PEG-15 cocamine sulfate",
                 "Dioctyl sodium sulfosuccinate",
                 "sodium coco sulfate",
-                "Sulfate"
+                "Sulfate",
+                   "Alkylbenzene sulphonates",
+                "Alkyl Benzene Sulphonate",
+                "Ammonium laureth sulphate",
+                "ammonium dodecyl sulphate",
+                "Ammonium lauryl sulphate",
+                "Sodium laureth sulphate",
+                "Sodium lauryl sulphate",
+                "Sodium myreth sulphate",
+                "TEA-dodecylbenzenesulphonate",
+                "Ethyl PEG-15 cocamine sulphate",
+                "sodium coco sulphate",
+                "Sulphate"
```

Comment:
Your description says to add `Sulphate` but there are a bunch of other new ingredients. Did you mean to do that>?

Reply:
yes